lib/phone_shop/widgets/components/availability_card.dart

```diff
@@ -0,0 +1,104 @@
+import 'package:flutter/material.dart';
+
+class AvailabilityCard extends StatelessWidget {
+  final String category;
+  final int stock;
+
+  const AvailabilityCard({
+    super.key,
+    required this.category,
+    required this.stock,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return Card(
+      color: const Color(0xFFFFFFFF),
+      shape: RoundedRectangleBorder(
+        borderRadius: BorderRadius.circular(12),
+        side: const BorderSide(
+          color: Color(0xFFF1F1F1),
+          width: 1,
+        ),
+      ),
+      child: Padding(
+        padding: const EdgeInsets.all(16),
+        child: Column(
+          children: [
+            Row(
+              mainAxisAlignment: MainAxisAlignment.spaceBetween,
+              children: [
+                const _CategoryText(),
+                const SizedBox(width: 10),
+                _Category(category: category),
```

Comment:
Same as in previous, on small screen break Category in more lanes do not leave overflow